tests/playwright-test/babel.spec.ts

```diff
@@ -36,3 +36,43 @@ test('should succeed', async ({ runInlineTest }) => {
   expect(result.passed).toBe(1);
   expect(result.failed).toBe(0);
 });
+
+test('should treat enums equally', async ({ runInlineTest }) => {
+  test.fail(true, 'https://github.com/microsoft/playwright/issues/13265');
+  const result = await runInlineTest({
+    'component.tsx': `
+      export enum MyEnum {
+        Value = "value",
+      };
+
+      export const enum MyConstEnum {
+        Value = "value",
+      }
+    `,
+    'regular.ts': `
+      export enum MyEnum {
+        Value = "value",
+      };
+
+      export const enum MyConstEnum {
+        Value = "value",
+      }
+    `,
+    'example.spec.ts': `
+      const { test } = pwt;
+
+      import * as components from './component';
+      import * as regular from './regular';
+
+      test('works', () => {
+        expect.soft(components.MyEnum.Value).toBe("value");
+        expect.soft(components.MyConstEnum.Value).toBe("value");
```

Comment:
@pavelfeldman What should we expect `components.MyConstEnum.Value` to be? It's currently "undefined", while those coming out of `regular.ts` (below) are defined as "value".

Reply:
We don't compile TSX, we only stub those altogether for component testing. So this test documented present behavior.